examples/plugin/CMakeLists.txt

```diff
@@ -0,0 +1,24 @@
+cmake_minimum_required(VERSION 3.2)
+
+project(example-plugin)
+
+include(GNUInstallDirs)
+
+find_package(f3d REQUIRED)
+
+f3d_plugin_init()
+
+f3d_plugin_declare_reader(
+  NAME Example
+  EXTENSIONS expl
+  MIMETYPES application/vnd.expl
+  VTK_READER vtkF3DExampleReader
+  DESCRIPTION "Example reader"
+)
+
+f3d_plugin_build(
+  NAME example
+  VERSION 1.0
+  DESCRIPTION "Example plugin"
+  VTK_MODULES CommonCore
+)
```

Comment:
add a test ?

Reply:
I thought about it, but isn't it overkill? We don't care about examples coverage, it's just there to make sure the SDK is working well, and giving usage examples to the users.

Reply:
its not about coverage, it's about checking that the plugin works.